common/djangoapps/track/middleware.py

```diff
@@ -36,6 +36,29 @@
 }
 
 
+class ResponseLoggingMiddleware(object):
+    """
+    A debugging middleware for the purpose of understanding what the current state of the response
+    is at this point in the stack.
+    """
+
+    def process_response(self, response):
```

Comment:
Shouldn't this take the request as a parameter also?  Docs for `MIDDLEWARE_CLASSES`-style middleware: https://docs.djangoproject.com/en/1.9/topics/http/middleware/#process-response

Reply:
Thanks for catching that!